shared/src/main/java/org/geysermc/hydraulic/pack/context/PackContext.java

```diff
@@ -67,12 +69,22 @@ public T module() {
      * @param <V> the type of the registry
      */
     @NotNull
-    public <V> List<V> registryValues(@NotNull ResourceKey<Registry<V>> key) {
+    @SuppressWarnings("RedundantCast")
```

Comment:
Why do we need to suppress this?

Reply:
Because IntelliJ thinks the casts to `RegistryKey<?>` are redundant, but they are not.